src/live_restore/live_restore_fs.c

```diff
@@ -1622,11 +1682,15 @@ __wt_os_live_restore_fs(
     WT_ERR(__wt_config_gets(session, cfg, "live_restore.threads_max", &cval));
     lr_fs->background_threads_max = (uint8_t)cval.val;
 
-    __wt_verbose_debug1(session, WT_VERB_LIVE_RESTORE,
-      "WiredTiger started in live restore mode! Source path is: %s, Destination path is %s",
-      lr_fs->source.home, destination);
+    /* Configure the read size. */
+    WT_ERR(__wt_config_gets(session, cfg, "live_restore.read_size", &cval));
+    lr_fs->read_size = (uint64_t)cval.val;
+
+    printf(
```

Comment:
At this stage the verbose system isn't configured yet, I'm not a fan of a printf either really, any recommendations?

Reply:
Yes it is AFAICT. In `wiredtiger_open` the call to `wt_verbose_config` is called at line 2972 and you added the call to copy the log files at line 3130.

Reply:
This is part of the live restore FS initialization which is called at wiredtiger_open:2875.

Reply:
05e7091d5